docs/conf.py

```diff
@@ -54,16 +56,16 @@
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 #
-# source_suffix = ['.rst', '.md']
-source_suffix = ".rst"
+# SOURCE_SUFFIX = ['.rst', '.md']
+SOURCE_SUFFIX = ".rst"
 
 # The master toctree document.
-master_doc = "index"
+MASTER_DOC = "index"
 
 # General information about the project.
-project = "pynautobot"
-copyright = "TBD"  # "2017, DigitalOcean"
-author = "Network to Code"  # "Zach Moody"
+PROJECT = "pynautobot"
+COPYRIGHT = "TBD"  # "2017, DigitalOcean" # pylint: disable=W0622
```

Comment:
```suggestion
COPYRIGHT = "TBD"  # "2017, DigitalOcean"
```

Now that's it's all caps, it no longer shadows the builtin.

Reply:
Updated.